src/Helpers/HtmlToMarkdown.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Bonnier\WP\ContentHub\Editor\Helpers;
+
+use Exception;
+
+class HtmlToMarkdown extends \GuzzleHttp\Client
+{
+    const ENDPOINT = 'http://html-to-markdown.interactives.dk';
+
```

Comment:
Could this be used instead? https://github.com/thephpleague/html-to-markdown

Reply:
No because it does not support GFM, which is why we have made that service